tools/vsimporter/src/vsimporter.cpp

```diff
@@ -282,15 +290,11 @@ int main(int argc, char* argv[])
   // Initialize global settings
   String binaryDir = sb_dirname(getBinaryLocation());
   sbValidateWithTelemetry(!binaryDir.empty(), "Failed to resolve binary directory.");
+  if (!templatesPath.empty()) {
+    settingsManager.setGlobalVar("VSIMPORTER_TEMPLATES_DIR", joinPaths(getcwd(), templatesPath));
```

Comment:
>getcwd() [](start = 71, length = 8)

don't do this. Its a bad idea to assume that the cwd is the vsimporter directory. I think this should be binaryDir with some relative path?